config-overrides.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
         enabled: env === 'production',
         hashEnabled: { 'style-src': false },
         nonceEnabled: { 'style-src': false }
-      }))
+      })),
+      _.update('ignoreWarnings', (old = []) => _.concat(old, /Failed to parse source map/))
```

Comment:
Why was this necessary? Asking for my edification.

Reply:
Try removing it and running locally and watch the output